src/Release/UseCase/CreateRelease/Response.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * @author: Ryan Wood <ryanw@clearbooks.co.uk>
+ * @created: 16/07/15
+ */
+
+namespace Clearbooks\Labs\Release\UseCase\CreateRelease;
+
+
+interface Response
+{
+    const INVALID_NAME_ERROR = 'Invalid Argument, the release must have a non empty string for a release name';
```

Comment:
Not sure I'd have the actual text to show here, converting an error const to a human readable string is very much the responsibility of the presenter